documentation/navitia.io/source/integration.rst

```diff
@@ -276,6 +276,27 @@ Coverage (/coverage)
 You can easily navigate through regions covered by navitia.io, with the coverage api.
 The only arguments are the ones of `paging`_.
 
+Inverted geocoding (/coord)
+***************************
+In order to determinate the right coverage, you can request navitia with a WGS84 coordinate as:
```

Comment:
I think we should clearly speak about both uses:
* Inverted geocoding (with explaination)
* coverages list